### ISSUE TITLE
add swagger support

### ISSUE THREAD

**Issue by FedorRomanov** [tomtom-international/configuration-service#2]
[Swagger](http://swagger.io), a tool for representation of RESTful API, can be added to configuration-service. It will allow to test configuration-service from pretty looking html gui and expose documentation to the clients.

**Comment by rijnb** [comment]
Indeed possible. But that's really why I've added rather extensive examples in the README.md file, so developers would get the hang of it.
By the way, did you notice you get the full help text if you call GET / (the root node), as HTML? 
(The README does not document all calls, such as /version, /status and /.)

**Comment by FedorRomanov** [comment]
Yes, README.md is quite detailed and have well-described examples. However, the beauty of swagger that it sustains live html documentation - it is generated from swagger annotations during build time. So the docs will never gets outdated(as it usually happens after a while).

Once I peeked into `RootResource.java`, I've discovered `String getHelpHTML()` service. As I mentioned in [another issue](https://github.com/tomtom-international/configuration-service/issues/3) describing helper services(health check, displaying version, displaying help) should be mentioned in main README.md

**Comment by rijnb** [comment]
Good point. At the very least the `README` must be complete in that sense. I'll contact you later about the Swagger stuff (which funnily enough I introduced at our company myself years ago...).

**Comment by rijnb** [comment]
Marked as an enhancement.
